modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1322,21 +1311,87 @@ public void testBindObjectDefinitions() throws Exception {
 			new String[] {"C_A", "C_AA", "C_AAA", "C_AAB"});
 	}
 
+	@Test
+	public void testDeleteCompanyObjectDefinitions() throws Exception {
+		String originalName = PrincipalThreadLocal.getName();
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		Company company = CompanyTestUtil.addCompany();
+
+		PortalInstances.initCompany(company);
+
+		try (SafeCloseable safeCloseable =
+				CompanyThreadLocal.setWithSafeCloseable(
+					company.getCompanyId())) {
+
+			User user = UserTestUtil.getAdminUser(company.getCompanyId());
+
+			Assert.assertNotNull(user);
+
+			PermissionThreadLocal.setPermissionChecker(
+				PermissionCheckerFactoryUtil.create(user));
+
+			PrincipalThreadLocal.setName(user.getUserId());
```

Comment:
@seltong why do we have a empty line here

Reply:
I just followed some examples in the code, but this is a good question. I couldn't find out the pattern for this case.

Examples:

- [DDMFormInstanceStagedModelDataHandlerTest.java#L127-L130](https://github.com/liferay/liferay-portal/blob/c39f096973ec5991c0c748b07f43ad55a8187ac5/modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/exportimport/data/handler/test/DDMFormInstanceStagedModelDataHandlerTest.java#L127-L130)
- [BatchEngineBrokerTest.java#L600-L604](https://github.com/liferay/liferay-portal/blob/0e748e689423d40b9cbd049a4e3ce11d37252e9c/modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java#L600-L604)
- [UserAccountResourceTest.java#L1302-L1306](https://github.com/liferay/liferay-portal/blob/06734fe09a2f53c606cd38a1b27032360b0ecd4b/modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java#L1302-L1306)
- [ObjectEntryResourceTest.java#L3984-L3986](https://github.com/liferay/liferay-portal/blob/22811994968d82988e7374a9c43dc4233e60edd5/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java#L3984-L3986)
- [ObjectFolderLocalServiceTest.java#L162-L165]( https://github.com/liferay/liferay-portal/blob/84630f4b30f3673ea44783c4deb4c45ad002785b/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFolderLocalServiceTest.java#L162-L165)

Should I remove the empty line?

Reply:
I think it's better to keep some pattern in this PR. As the code in Master is already messed up, you can remove it in both places or keep it in both places 😕

Reply:
Done, thanks